core/src/price_finding/price_finder_interface.rs

```diff
@@ -50,6 +50,33 @@ impl InternalOptimizer {
     }
 }
 
+/// The optimization algorithm used by standard and best-ring solvers.
+#[derive(Clone, Copy, Debug)]
+pub enum OptModel {
+    Mip,
+    TwoStage,
+}
+
+impl FromStr for OptModel {
+    type Err = Error;
+    fn from_str(string: &str) -> Result<Self> {
+        match string {
+            "mip" => Ok(Self::Mip),
+            "two_stage" => Ok(Self::TwoStage),
+            _ => Err(anyhow!("optimization model does not exit")),
+        }
+    }
+}
```

Comment:
nit: This shouldn't be needed here since we aren't reading an optimization model from a string, but instead it is being determined by its solver type. Maybe this should be a configurable option? If so, we can leave this in and add the command line parsing stuff in a follow-up PR.

Reply:
Removed in 18450de